### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Site_Name link_Logging_Logs: NVDA is reading invisible content after “Refresh” button.

### ISSUE THREAD

**Issue by v-supupp** [Microsoft/IIS.WebManager#188]
**Environment Details:**
URL: https://manage.iis.net/webserver/modules
Browser Details: Google Chrome Version 68.0.3440.106

**Screen Reader:** NVDA

**WCAG ID Violated:** WCAG1.3.1

GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #ScreenReader; #C+E-H; #.NETWebsites; #Chrome; #NVDA; WCAG1.3.1

**Repro Steps:**
1.	Download and install "Microsoft IIS Administration" application.
2.	Enable Internet information Services.
3.	Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4.	Tab till Web Site and press enter on any one of the name link field
5.	Tab till Logging and press enter
6.	Validate whether screen reader invisible content or not

**Actual Result:**
NVDA is reading invisible content after “Refresh” button

**Expected Result:**
NVDA should not read invisible content after “Refresh” button

**User Impact:**
Screen Reader users finds it difficult in understanding the application if it is reading invisible content

**Attaching videos  for your reference:**
[Bug 188_NVDA.pptx](https://github.com/Microsoft/IIS.WebManager/files/2335564/Bug.188_NVDA.pptx)

**Attaching screenshot which contains the content NVDA is reading after Refresh button:**
![bug 188_nvda_refresh button](https://user-images.githubusercontent.com/42638744/44844005-5441af80-ac67-11e8-8e8a-888fd46117de.png)

**Comment by jhkimnew** [comment]
Fixed
